packages/drivers/odsp-driver/src/odspSnapshotParser.ts

```diff
@@ -18,7 +18,7 @@ import { ISnapshotContents } from "./odspUtils";
 function buildHierarchy(flatTree: IOdspSnapshotCommit): api.ISnapshotTree {
     const lookup: { [path: string]: api.ISnapshotTree } = {};
     // id is required for root tree as it will be used to determine the version we loaded from.
-    const root: api.ISnapshotTree = { id: flatTree.id, blobs: {}, commits: {}, trees: {} };
+    const root: api.ISnapshotTree = { id: flatTree.id, blobs: {}, trees: {} };
```

Comment:
It would be great to remove ISnapshotTree.id as well. I believe there is no usage in the client code.

Reply:
It seems src/odspDocumentStorageManager.ts uses it

Reply:
Yes, it seems to be used in tree fetching code in getVersions and caching it.